src/module/TaskManagement/src/TaskManagement/Service/TaskServiceFactory.php

```diff
@@ -1,28 +1,29 @@
-<?php
-
-namespace TaskManagement\Service;
-
-use Zend\ServiceManager\FactoryInterface;
-use Zend\ServiceManager\ServiceLocatorInterface;
-
-use Ora\TaskManagement\EventSourcingTaskService;
-use Ora\TaskManagement\TaskService;
-
-class TaskServiceFactory implements FactoryInterface 
-{
-    /**
-     * @var TaskService
-     */
-    private static $instance;
-    
-	public function createService(ServiceLocatorInterface $serviceLocator) 
-	{
-	    if(is_null(self::$instance)) 
-	    {
-			$eventStore = $serviceLocator->get('prooph.event_store');
-			$entityManager = $serviceLocator->get('doctrine.entitymanager.orm_default');
-            self::$instance = new EventSourcingTaskService($eventStore, $entityManager);            
-        }
-	    return self::$instance;
-	}
+<?php
+
+namespace TaskManagement\Service;
```

Comment:
Non capisco cosa sia cambiato in questo file. Se non ci sono modifiche non committare un file, ma ripristina la versione HEAD. Semplifichiamo i processi di review, di rebase e di merge non aggiungendo modifiche non legate a contenuti.